MinnPost/object-sync-for-salesforce
# 220 expand last pull modified date

## What does this PR do?

Fixes #220, in which pulling a batch of modified records in Salesforce could result in some of them being incorrectly skipped, if any of them were correctly not allowed to process.

## How do I test this PR?

1. Create a fieldmap that allows updated records to be pulled, but not created records.
2. Update a record that is already mapped in Salesforce so it should be updated in WordPress.
3. Create a new record in Salesforce that should not be created in WordPress.

All of these should behave as expected, if the fix works.
